### DISCUSSION TITLE
the calling link is discontinuous during load test

### DISCUSSION THREAD

**Discussion by renhaizhou** [apache/skywalking#7675] [Q&A] [answered]
When I configure agent.sample_n_per_3_secs = 1500 and receiver-trace.default.sampleRate = 100, the calling link is discontinuous during load test. Shall I restore this parameter to agent.sample_n_per_3_secs = = - 1

**Comment by wu-sheng** [comment, marked as answer]
`agent.sample_n_per_3_secs = 1500` doesn't affect the sampling usually, unless MQ consumer involved. Because in that case, sampling runs first than check whether there is existing tracing context. In most RPC cases, this is not an issue.

**Reply by renhaizhou** [reply]
If agent.sample_n_per_3_secs = 1500 is configured, will the same request link occur when the first service is sampled and the second service is not sampled

**Reply by wu-sheng** [reply]
The source codes are there, I checked, and confirmed my statements in the morning.
It is better to verify by yourself.

**Reply by renhaizhou** [reply]
After reading the source code, I feel that if the sampling rate is set, the same request link will be sampled through the first service. If the second service has reached the upper limit of 1500 within 3 seconds, it will not be sampled. Then create an IgnoredTracerContext. In this way, the same request link will be missing. I want to confirm with you that my analysis is correct.

**Reply by wu-sheng** [reply]
I don't think so. If the context exists in header, force sampling would be activated.

**Comment by renhaizhou** [comment]
I see. Thank you
